nixos/modules/services/monitoring/prometheus/exporters/collectd.nix

```diff
@@ -61,6 +61,7 @@ in
     collectSettingsArgs = if (cfg.collectdBinary.enable) then ''
       -collectd.listen-address ${cfg.collectdBinary.listenAddress}:${toString cfg.collectdBinary.port} \
       -collectd.security-level ${cfg.collectdBinary.securityLevel} \
+      ${if cfg.collectdBinary.authFile != null then "-collectd.auth-file ${cfg.collectdBinary.authFile}" else ""} \
```

Comment:
`optionalString (cfg.colledtdBinary.authFile != null) "-collextd.auth-file ${cfg.collectdBinary.authFile}"`

Reply:
Thanks for hint, I have fixed that.